app/views/posts/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="form_div">
+<%= form_with model: @post, url: user_posts_path do|form| %>
+   <div><%= form.text_field :title, placeholder: "Post Title" %></div>
+  <div>  <%= form.text_area :text, placeholder: "Post content" %></div>
+    <div><%= form.submit "Add Post" %></div>
+    <% end %>
+</div>
```

Comment:
- [ ] Good job so far, the project implementation looks good :heavy_check_mark: Unfortunately it is not possible to populate your database to make others see the forms created and match with the given Wireframe and all the functions work. Kindly check the screenshot below :arrow_down:  Please fix it in order to make your app work as expected :+1: 

![Screenshot from 2022-12-01 23-46-34](https://user-images.githubusercontent.com/6615394/205183288-6a85d594-1557-4581-a0da-34e7688d82bf.png)
![Screenshot from 2022-12-01 23-46-17](https://user-images.githubusercontent.com/6615394/205183292-d106b6f7-060f-4f23-a13e-ee7ed5dbd26f.png)

Reply:
Hi @codecaiine This is not an issue being presented on my end, so I will need more info from you about why this is happening.